docs/interface_jvmti.md

```diff
@@ -595,7 +595,7 @@ You can query VM log options by using the `QueryVmLogOptions()` API:
 
     jvmtiError QueryVmLogOptions(jvmtiEnv* jvmti_env, jint buffer_size, void* options, jint* data_size_ptr)
 
-This extension returns the current log options as an ASCII string. The syntax of the string is the same as the [`-Xsyslog`](xsyslog.md) command-line option, with the initial `-Xsyslog:` omitted. For example, the string "error,warn" indicates that the VM is set to log error and warning messages only. For more information, see [`-Xsyslog`](xsyslog.md).
```

Comment:
What's wrong with having links? It's easier to use a link than to lookup the option.

Reply:
We don't need 2 links to the same topic in the same paragraph. This is not ideal from a UX perspective.